packages/monaco/src/typings/monaco/index.d.ts

```diff
@@ -8,12 +8,47 @@ declare module monaco.instantiation {
 declare module monaco.editor {
 
     export interface ICommonCodeEditor {
+        readonly _codeEditorService: monaco.services.ICodeEditorService;
         readonly _commandService: monaco.commands.ICommandService;
         readonly _instantiationService: monaco.instantiation.IInstantiationService;
         readonly _contributions: {
             'editor.controller.quickOpenController': monaco.quickOpen.QuickOpenController
         }
         readonly cursor: ICursor;
+        setDecorations(decorationTypeKey: string, decorationOptions: IDecorationOptions[]): void;
```

Comment:
ok, now see why there is not `Disposable` :) so how one can remove by setting an empty array? Could you document on our API level?

Reply:
sure!